webrender/res/cs_clip_border.vs.glsl

```diff
@@ -85,15 +85,15 @@ void main(void) {
         switch (corner.clip_mode) {
             case CLIP_MODE_DASH: {
                 // Fetch the information about this particular dash.
-                BorderClipDash dash = fetch_border_clip_dash(cci.data_index + 2 + 2 * (cci.segment_index - 1));
+                BorderClipDash dash = fetch_border_clip_dash(cci.clip_data_address, 2 + 2 * (cci.segment - 1));
```

Comment:
nit: now that the offset has a separate semantics from the base address, it might be reasonable to move the segment -> offset computation logic into the corresponding `fetch_border_clip*` functions, so that here you only call it as `fetch_border_clip_dash(cci.clip_data.address, cci.segment)`